build.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+set -e
+
+bsdtar -Oxf sketcher.deb 'data.tar.*' |
+  bsdtar -xf - \
+    --strip-components=2 \
+    --exclude='./etc/' \
+    --exclude='./usr/bin/' \
+    --exclude='./usr/share/doc/'
+
+mv WireframeSketcher ${FLATPAK_DEST}/
+
+find share -type f -exec install -Dm644 "{}" "${FLATPAK_DEST}/{}" \;
+
+install -Dm644 flatpak-dev-shim.jar ${FLATPAK_DEST}/flatpak-dev-shim.jar
+install -Dm644 flatpak-dev-shim.so ${FLATPAK_DEST}/lib/libflatpakdevshim.so
+
+mkdir ${FLATPAK_DEST}/bin
+ln -s ${FLATPAK_DEST}/WireframeSketcher/WireframeSketcher ${FLATPAK_DEST}/bin/WireframeSketcher
+
+sed -i -e 's/<\/component>/\t<launchable type="desktop-id">wireframesketcherstudio.desktop<\/launchable>\n\t<content_rating type="oars-1.1"\/>\n<\/component>/g' ${FLATPAK_DEST}/share/appdata/wireframesketcherstudio.appdata.xml
```

Comment:
can you do this through a patch? If sed fails, the build will break.

Reply:
Do you mean a module source of type patch? I don't think I can use one with a deb file.

Reply:
why not ? `patch` works with any file, just do `mv /path/to/file/in/deb/foo.metainfo.xml /path/to/file/in/deb/foo.metainfo.xml.old`, then modify it and get the `diff` between two files

Reply:
Updated to use patch